synapse/daemon.py

```diff
@@ -429,24 +431,47 @@ def _loadSessConf(self, info):
 
             self.onfini( core.fini )
 
+    def _onTelePushFiniMesg(self, sock, mesg):
+        name = mesg[1].get('name')
+        guid = mesg[1].get('guid')
```

Comment:
it's probably fine, but i usually use iden to avoid coliding with a potential "from synapse.common import guid"